docs/packaging/Packaging using CMake.rst

```diff
@@ -31,9 +31,18 @@ A *.deb* package for Synfig can be made by using the following code.
 Windows
 -------
 
-A *.exe* installer for synfig can be made by using the following code.
+A *.exe* installer for Synfig can be made by using the following code.
 
 .. code:: bash
 
     $ cpack -G NSIS
 
+macOS (formely OSX)
+-------
+
+A *.app* installer for Synfig can be made by using the following code.
```

Comment:
But it can't.

Currently, the installer can only be made using the [3-package-osx-dmg.sh](https://github.com/synfig/synfig/blob/master/3-package-osx-dmg.sh) script.

This command can only be used to compile Synfig.

Reply:
Umm, yes, you are right, it finished but it didn't packed anything... 
Does [3-package-osx-dmg.sh](https://github.com/synfig/synfig/blob/master/3-package-osx-dmg.sh) script takes care of Autobuild and Cmake or just Autobuild?

Reply:
Currently autotools only